README.md

```
@@ -21,41 +21,42 @@
+| <a href="https://github.com/jinhyo0"><img src="https://avatars.githubusercontent.com/u/150879545?v=4" width="120px;" alt=""/></a> | <a href="https://github.com/seongmin36"><img src="https://avatars.githubusercontent.com/u/202721995?v=4" width="120px;" alt=""/></a> | <a href="https://github.com/onebone"><img src="https://avatars.githubusercontent.com/u/3233503?v=4" width="120px;" alt=""/></a> | <a href="https://github.com/codmoni"><img src="https://avatars.githubusercontent.com/u/160315926?v=4" width="120px;" alt=""/></a> |
+| --------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| 김진효                                                                                                                            | 조성민                                                                                                                               | 정윤철                                                                                                                          | 황무원                                                                                                                            |
 
 </div>
 
-<br/> 
+<br/>
 
 <div>
   
   ## 💻 기술 스택
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)